cardstack/src/screens/DepotScreen.tsx

```diff
@@ -76,174 +106,110 @@ export default function DepotScreen() {
+        paddingHorizontal={5}
+        justifyContent="space-between"
+      >
+        <CenteredContainer flexDirection="row" justifyContent="space-between">
+          <Icon
+            color="teal"
+            iconSize="medium"
+            name="chevron-left-no-box"
+            onPress={goBack}
+          />
+          <Container alignItems="center">
+            <Text color="white" weight="bold" textAlign="center">
+              Depot
+            </Text>
+            <Touchable
+              onPress={onPressInformation}
+              flexDirection="row"
+              alignItems="center"
+            >
+              <Text
+                fontFamily="RobotoMono-Regular"
+                color="white"
+                size="xs"
+                marginRight={2}
+              >
+                {getAddressPreview(address)}
+              </Text>
+              <Icon name="info" size={15} />
+            </Touchable>
           </Container>
+          <Icon name="more-horizontal" color="teal" onPress={onPressMore} />
+        </CenteredContainer>
+      </Container>
+      <TabHeader />
+      <Container flex={1} width="100%" backgroundColor="white">
+        <AbsoluteFullScreenContainer
+          flex={1}
+          backgroundColor="white"
+          zIndex={getTabStackOrder(Tabs.BALANCES)}
+        >
+          <BalanceSection tokens={tokens} safeAddress={address} />
+        </AbsoluteFullScreenContainer>
+        <Container
+          flex={1}
+          backgroundColor="white"
+          zIndex={getTabStackOrder(Tabs.ACTIVITIES)}
+        >
+          {shouldRenderActivities && <Activities depotAddress={address} />}
         </Container>
       </Container>
-    </Container>
+    </SafeAreaView>
   );
 }
 
-interface TabHeaderProps {
-  setSelectedTab: (tab: Tabs) => void;
-  selectedTab: Tabs;
-  tab: Tabs;
-}
-
-const TabHeader = ({ tab, selectedTab, setSelectedTab }: TabHeaderProps) => {
-  const isSelected = selectedTab === tab;
-  return (
-    <Touchable
-      alignItems="center"
-      justifyContent="center"
-      width="40%"
-      paddingTop={5}
-      onPress={() => setSelectedTab(tab)}
-    >
-      <Text
-        color="white"
-        marginBottom={3}
-        weight={isSelected ? 'bold' : 'regular'}
-      >
-        {tab}
-      </Text>
-      <Container
-        backgroundColor={isSelected ? 'teal' : 'transparent'}
-        height={4}
-        width="100%"
-      />
-    </Touchable>
-  );
-};
-
-const Activities = () => {
-  const {
-    params: { depot },
-  } = useRoute<RouteType<Params>>();
-
-  const { address } = depot;
-
+const Activities = React.memo(({ depotAddress }: { depotAddress: string }) => {
   const {
     sections,
     isFetchingMore,
     onEndReached,
     refetchLoading,
     refetch,
     isLoadingTransactions,
-  } = useDepotTransactions(address);
+  } = useDepotTransactions(depotAddress);
 
   return (
     <Container marginTop={7} flexDirection="column" width="100%">
```

Comment:
```suggestion
    <Container flexDirection="column" width="100%">
```
This margin adds a gap that is apparent when scrolling. I suggest removing this margin and adding `paddingTop: 20` contentContainerStyle.

Ideally we would add this paddingTop to layouts.ts -> contentContainerStyle, but then all components using that will need to be checked and updated. We could add a ticket to handle that too if you prefer.

Reply:
That's just a leftover from the previous implementation, the only change I made was the loading, and the props, but I'll take a look this gapping

Reply:
This "gap" seems to only happen on Android, iOS has stickyHeader as default whereas Android doesn't, so I'm enabling it, to have the same behavior

Reply:
https://user-images.githubusercontent.com/20520102/186183613-fd795924-77c3-45f8-b28e-cdb8cad1908f.mp4